update-version

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -e
+
+# Whitespace-separated list of components of the form 'service|image|tag'.
+COMPONENTS="\
+    flow-manager|gpii/universal|latest \
+    preferences-dataloader|gpii/preferences-dataloader|latest \
+    preferences|gpii/universal|latest \
+"
+
+for component in $COMPONENTS ; do
+    service=$(echo "$component" | awk -F'|' '{print $1}')
+    image=$(echo "$component" | awk -F'|' '{print $2}')
+    tag=$(echo "$component" | awk -F'|' '{print $3}')
+    echo "Looking up '$image:$tag' for service '$service'..."
+    api_token=$(\
+        curl -L -s \
+        "https://auth.docker.io/token?service=registry.docker.io&scope=repository:$image:pull" \
+        | jq -r .token \
```

Comment:
I have used a retry parameters with curl to react quickly to transient issues: https://github.com/GPII/universal/blob/master/scripts/vagrantCloudBasedContainers.sh#L55

I'm not sure how often this is scheduled so suggesting it here in case it makes sense.

Reply:
Good idea. Stole flags from your link :)